Dockerfile

```diff
@@ -0,0 +1,18 @@
+FROM ruby:2.4.1-alpine
+
+ENV RUNTIME_PACKAGES="tini" \
+    BUILD_PACKAGES="build-base sqlite-dev" \
+    GEMSTASH_VERSION="1.0.4"
```

Comment:
The current latest version is [1.1.0](https://github.com/bundler/gemstash/blob/master/CHANGELOG.md#110-2017-07-31).